mlflow/pyfunc/backend.py

```diff
@@ -8,13 +12,78 @@
 from mlflow.models.container import ENABLE_MLSERVER
 from mlflow.pyfunc import ENV, scoring_server, mlserver
 
-from mlflow.utils.conda import get_or_create_conda_env, get_conda_bin_executable, get_conda_command
+from mlflow.utils.conda import (
+    get_or_create_conda_env,
+    get_conda_bin_executable,
+    get_conda_command,
+)
 from mlflow.tracking.artifact_utils import _download_artifact_from_uri
 from mlflow.utils.file_utils import path_to_local_file_uri
+from mlflow.utils.environment import _REQUIREMENTS_FILE_NAME, _PYTHON_ENV_FILE_NAME, _PythonEnv
 from mlflow.version import VERSION
 
 _logger = logging.getLogger(__name__)
 
+_MLFLOW_HOME_ENV_VAR = "MLFLOW_HOME"
+
+
+def _is_windows():
+    return os.name == "nt"
+
+
+def _join_commands(*commands):
+    sep = " & " if _is_windows() else " && "
+    return sep.join(commands)
+
+
+def _get_entrypoint():
+    return ["cmd", "/c"] if _is_windows() else ["bash", "-c"]
```

Comment:
Question:
Must we add "bash -c " / "cmd /c" for executing command ? Without adding it, what error will happen ?

Reply:
> Must we add "bash -c " / "cmd /c" for executing command

No, but we need it when we want join multiple commands with `&&` (or `&` for windows).